internal/converter/internal/otelcolconvert/testdata/otelcol_without_validation/sigv4auth.river

```diff
@@ -0,0 +1,29 @@
+otelcol.auth.sigv4 "default" {
+  	region  = "ap-southeast-1" 
```

Comment:
fyi: There's a formatting error here that isn't reported on yet mainly because of the test_common change :)

Reply:
Thanks! Fixed in HEAD now :D